src/app/meri-pehchaan/meri-pehchaan.component.html

```diff
@@ -0,0 +1,45 @@
+<div class="container-fluid pt-2 pb-2">
+    <div class="d-flex mx-3 mt-3">
+      <div>
+        <img src="assets/images/headerlogo.png" class="img-fluid" alt="...">
+      </div>
+      <div>
+        <h5 class="card-title fs-14 font-600">
+          <span class="text-gray-0 fs-12">DEPARTMENT OF</span> <br>
+          <span class="fs-16"> Basic Education</span><br>
+          <span class="text-orange fs-10">GOVERNMENT OF UTTAR PRADESH</span>
+        </h5>
+      </div>
+    </div>
+  </div>
+  <div id="sect1" class="container">
+    <div>
+      <h1 class="text-center page-title">
+        <span class="text-yellow">ULP </span>
+        <span class="text-blue">Registration Portal</span>
+      </h1>
+      <hr class="horizontal-line" />
+ 
+      <div class="text-center mt-5 pt-5 d-flex jc-content-center">
+        <div class="square pt-4"><img src="assets/images/profile.png" alt="">
+          <div class="py-2 pt-3 fs-16 font-600"> {{'PRINCIPAL' | translate }}</div>
+        </div>
+      </div>
+    </div>
+    <div class="d-flex justify-content-center mt-4">
+        <button (click)="openSSO('login-auth-principal')" type="button" class="custom-button blue-button mt-20 pr-90 fs-16" >Login using Meri Pehchaan</button>
```

Comment:
@shreyas1434shinde translate this as well

Reply:
@itsvick done